src/modules/tree.ts

```diff
@@ -488,4 +488,23 @@ export default {
       requiredArgs: 'edition',
     },
   },
+  logs: {
+    description: 'Show logs of an app on the current directory or a specified one',
+    handler: './apps/logs',
+    optionalArgs: ['vendor', 'app'],
```

Comment:
Prefer using flags for this - it becomes much clearer for the user, for example:
```
vtex logs --vendor storecomponents
vtex logs --app vtex.builder-hub@0.x
```